### ISSUE TITLE
v14 => v15 Error: System.Exception: Module upgrade did not complete

### ISSUE THREAD

**Issue by eXistenZe** [2sic/2sxc#3047]
### I'm submitting a ...
[x] bug report 

### ...about
[x] admin experience UI
[x] other / unknown

### Instructions to Reproduce the Problem
v14.11 install working fine on a local DNN 9.11.1 install.

Upgrading to 15.4 generates the 
"Error: System.Exception: Module upgrade did not complete (read more). Click to complete"
But clicking does nothing because it generates 

https://site/api/2sxc/sys/install/resume
500 Internal Server Error

### Your environment

* **Hosting platform:** Windows server 2022

**Comment by iJungleboy** [comment]
Based on this information we cannot help you.
So far quite a few websites have been upgraded so it seems to work, but a 500 error on your system could be anything. Pls check the insights as they may have a suggestion what's happening. Maybe files are locked or something.

**Comment by iJungleboy** [comment]
APIs are never real files - they are `endpoints` which the system takes care of. All 2sxc endpoints look a bit like this, but can vary depending on the version of DNN.

Insights: https://docs.2sxc.org/net-code/debug/insights.html?q=insights

**Comment by eXistenZe** [comment]
Ok. No more permission issues. IIS app pool has write access to the files (even full). Sql server is fine. Files are unblocked, I can modify the files through dnn (config manager, resource manager, adam, etc).

Upgrading 2sxc results in this error nevertheless.
Insight logs shows nothing I can see as usefull. Are they stored somewhere I can share?

**Comment by iJungleboy** [comment]
Note that we did find some cases where file locking could be an issue. 
Best try 15.06 to be released today.
Also check the `DesktopModules\ToSIC_SexyContent\Upgrade\Log` and review the latest files there.

**Comment by iJungleboy** [comment]
I'll close this for now, as I believe it doesn't affect others.
If you have further issues just post them here, as I'll get notified.

**Comment by eXistenZe** [comment]
I tried before the latest v14 lts and then v15.0 and it upgraded with no issues. 

15,4 and 15.5 failed, so, as recommended, I checked the update logs and the red flag was obvious:
2023-03-29T13:59:26 15.02.00 - Upgrade failed - The transaction log for database 'db' is full due to 'ACTIVE_TRANSACTION' and the holdup lsn is (1109:136:212).

15.4 and 15.5 apparently bumps the database size in over 40mb and exceeded the db size limit. Fixed now.
